build.sbt

```diff
@@ -391,7 +391,7 @@ docker / dockerfile := {
   val readme = rootDir / "README.md"
 
   new Dockerfile {
-    from("eclipse-temurin:17")
+    from("azul/zulu-openjdk:17-latest")
```

Comment:
I think we recommend temurin in the docs?
Why switch?

Reply:
For some reason, the Temurin image stopped installing sudo. Perhaps, they have removed the package, idk.

Reply:
e.g., see  this: https://github.com/apalache-mc/apalache/actions/runs/10401320017/job/28803618698#step:5:190

Reply:
From the log, it looks like `apt install` is asking for confirmation.

I think you can simply add `-y` in line 409 ?

Reply:
Ah, you are right! It does work this way. I will revert it back to Temurin and add `-y`